travis.sh

```diff
@@ -179,8 +179,10 @@ script () {
             true
             ;;
         system2)
-            ${TEST_DRIVER} ocaml | tail -n256
-            expr $PIPESTATUS && false
+            ${TEST_DRIVER} ocaml | tail -n1000
+            X=$PIPESTATUS
```

Comment:
It seems to work with just $PIPESTATUS, so I think in this case they are equivalent.